src/AppRoutes.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Routes, Route, Navigate, BrowserRouter } from "react-router-dom";
+import LandingPage from "./pages/LandingPage";
+import HomePage from "./pages/HomePage";
+import LoginPage from "./pages/LoginPage";
+import { useContext } from "react";
+import { UserContext } from "./contexts/UserContext";
+
+function AppRoutes() {
+  const { userInfo } = useContext(UserContext);
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route
```

Comment:
You might want to add a * route for handling undefined paths, possibly redirecting to a "Not Found" page or the homepage.

Reply:
As suggested, I added the *route